JavaScript/chapter01/1.7 - Rotate Matrix/rroque98_sol.js

```diff
@@ -0,0 +1,25 @@
+/*Given an image represented by an N*N matrix where
```

Comment:
good PR ettiquette dictates that you should try to avoid fixing stuff _and_ adding new things into the same PR:

* https://medium.com/@hugooodias/the-anatomy-of-a-perfect-pull-request-567382bb6067
* https://www.kernel.org/doc/html/v4.17/process/submitting-patches.html#separate-your-changes
* https://medium.com/@fagnerbrack/one-pull-request-one-concern-e84a27dfe9f1

do you mind moving this to a separate PR? that way if one PR gets a lot of comments and the other one does not, you still do net progress since you'll be able to land one faster.